### ISSUE TITLE
Inconsistent behaviour with workspaces

### ISSUE THREAD

**Issue by jeffrson** [lerna-lite/lerna-lite#492]
### Describe the bug

lerna-lite does not recognize packages if they're not in "packages" (but in "apps" for example) as long as there's no 'useWorkspaces: true' in lerna.json. This property is not added by `lerna init --use-workspaces` and it seems undocumented (I just have it because I'm migrating from lerna 4).

OTOH, if I just change "packages/*" in "workspaces" in package.json to "apps/*" but don't actually rename the folder, packages in "packages" are still found.

### Expectation

* search packages as specified by workspaces property in package.json
- ideally without "useWorkspaces: true" in lerna.json, but
- if it's necessary add it through `lerna init --use-workspace` and
- mention this in the docs.

Thank you for taking over lerna!!!

### Reproduction

* `lerna init --use-workspaces`
* mkdir packages/p1 and create a basic package.json there
* change "packages/*" in package.json into "apps/*"
* try lerna list - still found, though "apps" does not exists
* rename packages to apps
* try lerna list - nothing found
* put "useWorkspaces: true" into lerna.json
* try lerna list - workspace is found again
* 

### Environment Info

@lerna-lite: 1.15.1

Happens with npm, yarn1, yarn3

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/lerna-lite/lerna-lite/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://github.com/lerna-lite/lerna-lite#readme).
- [X] Check that there isn't [already an issue](https://github.com/lerna-lite/lerna-lite/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/lerna-lite/lerna-lite/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ghiscoding** [comment]
Sorry but I'm not sure to understand what you're asking here. Is it just documentation problems that you have or is it something else? 

Typically, `useWorkspaces` was originally created for Yarn when `workspace` is defined in the `package.json`, later on the npm package manager also added this feature as per their docs [npm `workspaces`](https://docs.npmjs.com/cli/v7/using-npm/workspaces#defining-workspaces), when you `lerna init --use-worspaces` it will simply enable the flag in `lerna.json`. 
If you are not enabling `useWorkspaces`, it will instead use the `packages` defined in `lerna.json`. As for the lerna list command, I'm actually not sure if uses or care about `useWorkspaces` flag, maybe not I don't know since all I did was to copy over what Lerna had already implemented themselves. 

I think `lerna init --use-workspaces` is already clear enough, it does show what the result will be
https://github.com/lerna-lite/lerna-lite/tree/main/packages/init#--use-workspaces

lerna.json
```json
{
  "version": "0.0.0"
}
```
package.json (at the project root)
```json
{
  "name": "monorepo",
  "devDependencies": {
    "@lerna-lite/cli": "^1.0.0"
  },
  "workspaces": ["packages/*"]
}
```

the default folder for the packages will always be `packages/`, if you want to use something else it's totally up to you but you have to update it yourself either in `lerna.json` (without `useWorkspaces`) or `package`.json` to point to the correct folder(s)

**Comment by jeffrson** [comment]
Thank you for your detailed answer!

And I'm sorry if I wasn't clear enough. However, I didn't know for sure how it is supposed to work, so I tried to cover different aspects.

Eventually I believe that all this confusion could be solved, if `lerna init --use-workspaces` actually puts `"useWorkspaces": true` into lerna.json. Furthermore, this "useWorkspaces" is not part of the JSON schema (other than as command line parameter) und should be probably appended there as well.

**Comment by ghiscoding** [comment]
> Eventually I believe that all this confusion could be solved, if `lerna init --use-workspaces` actually puts `"useWorkspaces": true` into lerna.json.

Oh you're right I totally missed that, it looks like I only added the `workspace` section in the `package.json` but actually forgot to add `useWorkspaces` into `lerna.json`, my bad. I'll fix that later today and I'll also review the schema afterward for the init command, thanks for pointing that out. 

Also just so you know, there's more info in this [`lerna.json`](https://github.com/lerna-lite/lerna-lite/wiki/lerna.json) Wiki (which is linked in the homepage readme). The `useWorkspaces` is not mentioned in there yet but I think it would be a good idea to add the similar info that I wrote from my earlier [post](https://github.com/lerna-lite/lerna-lite/issues/492#issuecomment-1450818608). Typically speaking, if you come from Lerna v1-5, it should be just a drop-in replacement, the config are the same but with v6 they started to add Nx specific stuff that I won't replicate in Lerna-Lite
